docs/intro/tool-fundamentals/packer.md

```diff
@@ -0,0 +1,49 @@
+---
+'title': 'Packer'
+---
+
+# Packer
+
+[Packer](https://www.packer.io) is an open source tool you can use to define _machine images_ (e.g., VM
+images, Docker images) as code. For example, here is how you can use Packer to define an Ubuntu 18.04 Amazon Machine Image (AMI) that has Node.js installed:
+
+```json
+{
+  "builders": [{
+    "type": "amazon-ebs",
+    "region": "us-east-2",
+    "source_ami": "ami-0c55b159cbfafe1f0",
+    "instance_type": "t2.micro",
+    "ssh_username": "ubuntu",
+    "ami_name": "packer-example-{{timestamp}}"
+  }],
+  "provisioners": [{
+    "type": "shell",
+    "inline": [
+      "curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -",
+      "sudo apt-get update -y",
+      "sudo apt-get install -y nodejs"
+    ]
+  }]
+}
+```
+
+You can run `packer build packer-example.json` to build an AMI from this code and then deploy this AMI to your AWS
```

Comment:
I think this sentence is a little ambiguous. I think you're trying to say something like: "Here's how you use packer to build the AMI and then you can use our other modules to deploy servers that are imaged with the AMI you just built" - but I'm getting tripped up on:

> deploy this AMI to your AWS account using other tools.

To me, this is making it seem like the AMI _itself_ needs to be deployed. Ie - I have to use a different tool to upload the AMI to my AWS account. Consider rephrasing? Or maybe I'm just tired and not understanding :-D

Reply:
I've taken a crack at it in b4cca78.